secretstores/local/file/filestore_test.go

```diff
@@ -3,7 +3,9 @@ Copyright 2021 The Dapr Authors
 Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
-    http://www.apache.org/licenses/LICENSE-2.0
+
+	http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
For some reason this is what the linter wants on several files. And the VS Code Go extension made this modification automatically via `gofumpt`.

Reply:
I know why this is happening. Go 1.19 is adding markdown-ish support to comments. It should be ok